geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/v1/ProtobufSerializationService.java

```diff
@@ -38,6 +38,9 @@ public ProtobufSerializationService() {}
   @Override
   public BasicTypes.EncodedValue encode(Object value) throws EncodingException {
     BasicTypes.EncodedValue.Builder builder = BasicTypes.EncodedValue.newBuilder();
+    if (value == null) {
```

Comment:
Is this how we'll support null values going forward?

Reply:
I'm also curious about this, I thought we had decided that we shouldn't allow null values (in fact we removed the explicit null that we previously had)?

Reply:
No. That will need to be fixed in GEODE-4414.